lib/string_reverse.rb

```diff
@@ -1,6 +1,17 @@
 # A method to reverse a string in place.
-# Time complexity: ?
-# Space complexity: ?
+# Time complexity: O(n)

```

Comment:
Two questions:
- **Time complexity:** what is `n` here?
- **Space complexity:** what is this constant relative to?